src/components/text-field/react/TextField.tsx

```diff
@@ -48,6 +48,12 @@ interface ITextFieldProps extends IGenericProps {
     /** Theme. */
     theme?: Theme;
 
+    /** Line height of the input or text area element. */
+    lineHeight?: number;
```

Comment:
Should this be replaced with the minimum number of rows since you now calculate the line height dynamically?

Reply:
yep, forgot to deprecate that one.